plugins/modules/eos_ntp_global.py

```diff
@@ -0,0 +1,1000 @@
+#             }
+#         ],
+#         "local_interface": "Ethernet1",
+#         "qos_dscp": 10,
+#         "serve": {
+#             "access_lists": [
+#                 {
+#                     "acls": [
+#                         {
+#                             "acl_name": "acl01",
+#                             "direction": "in"
+#                         }
+#                     ],
+#                     "afi": "ip"
+#                 },
+#                 {
+#                     "acls": [
+#                         {
+#                             "acl_name": "acl02",
+#                             "direction": "in"
+#                         }
+#                     ],
+#                     "afi": "ipv6"
+#                 }
+#             ]
+#         },
+#         "servers": [
+#             {
+#                 "burst": true,
+#                 "prefer": true,
+#                 "server": "10.1.1.1",
+#                 "vrf": "vrf01"
+#             },
+#             {
+#                 "key": 2,
+#                 "maxpoll": 15,
+#                 "server": "25.1.1.1",
+#                 "vrf": "vrf01"
+#             }
+#         ],
+#         "trusted_key": "23"
+#     },
+#     "invocation": {
+#         "module_args": {
+#             "config": null,
+#             "running_config": null,
+#             "state": "gathered"
+#         }
+#     }
+# }
+
+
+# using rendered:
+
+  - name: Render provided configuration
+    arista.eos.eos_ntp_global:
+      config:
+        authenticate:
+          enable: true
+        authentication_keys:
+          - id: 2
+            algorithm: "sha1"
+            encryption: 7
+            key: "123456"
+          - id: 23
+            algorithm: "md5"
+            encryption: 7
+            key: "123456"
+        local_interface: "Ethernet1"
+        qos_dscp: 10
+        trusted_key: 23
+        servers:
+          - server: "10.1.1.1"
+            vrf: "vrf01"
+            burst: True
+            prefer: True
+          - server: "25.1.1.1"
+            vrf: "vrf01"
+            maxpoll: 15
+            key: 2
+        serve:
+          access_lists:
+            - afi: "ip"
+              acls:
+                - acl_name: "acl01"
+                  direction: "in"
+            - afi: "ipv6"
+              acls:
+                 - acl_name: "acl02"
+                   direction: "in"
+      state: rendered
+    become: yes
+
+# Module Execution:
+# "rendered": [
+#         "ntp serve ip access-group acl01 in",
+#         "ntp serve ipv6 access-group acl02 in",
+#         "ntp authentication-key 2 sha1 7 ********",
+#         "ntp authentication-key 23 md5 7 ********",
+#         "ntp server vrf vrf01 10.1.1.1 burst prefer",
+#         "ntp server vrf vrf01 25.1.1.1 key 2 maxpoll 15",
+#         "ntp authenticate",
+#         "ntp local-interface Ethernet1",
+#         "ntp qos dscp 10",
+#         "ntp trusted-key 23"
+#     ]
+#
+
```

Comment:
to update the RETURN in here

Reply:
Please re-install cli_rm_builder with `--force` to get latest changes that include the RETURN values boiler plate template.